Web/starter_project_barbies/public/data/dummy-data.json

```diff
@@ -0,0 +1,215 @@
+[
+    {
+      "userID": "user1",
+      "blogID": "1",
+      "title": "My First Blog Post",
+      "author": {
+        "name": "John Doe",
+        "email": "johndoe@gmail.com",
+        "image": "/img/blog-img/profile.jpg"
+      },
+      "tags" : ["UI/UX","Web Dev"],
+      "date": "2022-01-01",
+      "blogImage": "/img/blog-img/coding.jpg",
```

Comment:
i would recommend only saving the last part (because i assume `/img/blog-img` will be common for all blogs)
it gives you the flexibility to change the folder without needing to change the data